msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Integrate QR or barcode scanner capability
+description: How to use Teams JavaScript client SDK to enable QR or bar code scanner capability
+keywords: camera image microphone capabilities native device permissions media qr barcode scanner
+ms.topic: conceptual
+ms.author: lajanuar
+---
+
+# Integrate QR or barcode scanner capability 
+
+This document guides you on how to integrate QR or barcode scanner capability of the mobile phone with Teams platform. 
+You can read the code with an optical scanner feature of the native device camera. For richer collaborative experience, you can combine the scanner capability with the Teams platform through the integration process. 
+
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), that provides the tools necessary for your app to access the user’s [device permissions](native-device-permissions.md). After getting the camera access of the device, use **scanBarCode API** to integrate the scanner capability with Teams within your Microsoft Teams mobile app, and build a richer experience. 
+
+## Advantage of integrating QR or barcode scanner capability
+
+The main advantage of integrating QR or barcode scanner capability in your Teams apps is to help the developers on Teams platform to leverage QR or barcode scanning functionality with Teams JavaScript Client SDK.
+
+To integrate QR or barcode scanner capability, you must update the app manifest file and call the **`scanBarCode` API**. For effective integration, you must have a good understanding of [code snippets](#code-snippets) for calling the `scanBarCode` API, which allows you to use native QR or barcode scanner capability. The API throws error in case of an unsupported barcode standard.
```

Comment:
Please rephrase the following sentence:

 The API throws error in case of an unsupported barcode standard.